step/step.go

```diff
@@ -1,42 +1,72 @@
 package step
 
 import (
+	"encoding/json"
 	"fmt"
 	"os"
+
+	"github.com/stackpulse/steps-sdk-go/env"
 )
 
 const (
-	ExitCodeOK      = 0
-	ExitCodeFailure = 1
+	stepErrorKey = "step_error"
 )
 
 type Outputs struct {
 	Object interface{} `json:"api_object,omitempty"`
 }
 
+type JSONOutput map[string]interface{}
```

Comment:
```suggestion
type Output map[string]interface{}
```

why not just Output? JSONOutput is confusing since its not really json but a `map[string]interface{}`